bundler/spec/commands/remove_spec.rb

```diff
@@ -13,11 +13,10 @@
     end
   end
 
-  context "when --install flag is specified" do
+  context "when --install flag is specified", :bundler => "<3" do
     it "removes gems from .bundle" do
       gemfile <<-G
         source "#{file_uri_for(gem_repo1)}"
-
```

Comment:
Whitespace nitpick, let's leave it as it was. It gave me an idea of a rubocop cop, like `Bundler/BlankLineAfterGlobalSource` :)